mlxtend/regressor/stacking_cv_regression.py

```diff
@@ -163,6 +172,14 @@ def predict(self, X):
         else:
             return self.meta_regr_.predict(meta_features)
 
+    def predict_meta_features(self, X):
```

Comment:
Here, it would be helpful to format it as a docstring so that it shows up in the API documentation. E.g., something similar to the `fit` docstring. I just see that `predict`, `get_params` etc. don't have docstrings as well, but I can fix that later separately after this PR.